GOVERNANCE.md

```diff
@@ -0,0 +1,109 @@
+# Governance Model
+
+*This document is inspired by the [Meritocratic Governance Model] composed by
+OSS Watch. It should not be considered set in stone and may evolve over time as
+the project grows.*
+
+<!-- markdownlint-disable line-length -->
+[Meritocratic Governance Model]: http://oss-watch.ac.uk/resources/meritocraticgovernancemodel
+<!-- markdownlint-enable line-length -->
+
+## Overview
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project design
+and participate in the decision making process. This document describes how that
+participation takes place and how to set about earning merit within the project
+community.
+
+## Roles And Responsibilities
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor. There is no expectation of commitment
+to the project, no specific skill requirements and no selection process.
+
+In addition to their actions as users, contributors may also find themselves
+doing one or more of the following:
+
+* supporting new users (existing users are often the best people to support new
+  users)
+* reporting bugs
+* identifying requirements
+* providing graphics and web design
+* programming
+* assisting with project infrastructure
+* writing documentation
+* fixing bugs
+* adding features
+
+Contributors engage with the project through the issue tracker, writing or
+editing documentation, and participating in the [Oasis Community Slack
+channels]. They submit changes to the project itself via pull requests, which
+will be considered for inclusion in the project by existing committers (see next
+section).
+
+[Oasis Community Slack channels]: https://oasiscommunity.slack.com
+
+### Committers
+
+Committers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the
+community. The key difference between a committer and a contributor is when this
+approval is sought from the community. A committer seeks approval after the
+contribution is made, rather than before.
+
+Anyone can become a committer; there are no special requirements, other than to
+have shown a willingness and ability to participate in the project as a team
+player. Typically, a potential committer will need to show that they have an
+understanding of the project, its objectives and its strategy. They will also
+have provided valuable contributions to the project over a period of time.
+
+## Contribution Process
+
+Anyone can contribute to the project, regardless of their skills, as there are
+many ways to contribute. The [Oasis Community Slack channels] are the best place
+to ask questions and the Oasis Protocol GitHub is the best place to see open
```

Comment:
Looking at the doc again, it would make sense to also make "Oasis Protocol GitHub" a link for consistency.